src/api/app/models/project/exceptions.rb

```diff
@@ -0,0 +1,25 @@
+module Project::Exceptions
+  extend ActiveSupport::Concern
+  class CycleError < APIException
```

Comment:
Let's rename those `SomethingError` classes to `SomethingException` since they inherit from `APIException`

Reply:
I agree with that, ```APIException``` and ```CycleError``` don't match. However looks like in ruby exceptions are named with the suffix Error https://ruby-doc.org/core-2.2.0/Exception.html

So shouldn't we rename ```APIException``` to ```ApiError``` ?

Reply:
`APIError` is also fine for me.

Reply:
It inherited from Exception some time ago. We changed that [here](https://github.com/openSUSE/open-build-service/commit/2316f5b24f1e16fa02c1b227850ce2e61bb938a0). So the name does not match anymore. Yes rename it!